dbt/adapters/spark/connections.py

```diff
@@ -449,7 +449,7 @@ def open(cls, connection):
                         SessionConnectionWrapper,
                     )
 
-                    handle = SessionConnectionWrapper(Connection())
+                    handle = SessionConnectionWrapper(Connection(), creds.server_side_parameters)
```

Comment:
I would rather pass this in the `Connection` constructor.
```suggestion
                    handle = SessionConnectionWrapper(Connection(creds.server_side_parameters))
```
And then the connection can pass it down to the constructor of the `Cursor`. The config doesn't change on `.execute()` call, so I think it makes more sense to pass it in the constructors.

Reply:
@Fokko I haven't tested this, but is the new code what you had in mind?